Tests/Packet++Test/Makefile

```diff
@@ -51,7 +51,11 @@ dependents:
 
 Packet++Test: start create-directories $(OBJS_FILENAMES)
 	@cd $(MEMPLUMBER_HOME) && $(MAKE) memplumber_sources
+ifdef FREEBSD
+	@$(CXX) $(COMMON_DEBUG_LIB_DIR) $(PCAPPP_LIBS_DIR) $(PCAPPP_BUILD_FLAGS) -lexecinfo -o "./Bin/Packet++Test$(BIN_EXT)" $(OBJS_FILENAMES) $(MEMPLUMBER_OBJS_FILENAMES) $(PCAPPP_LIBS) 
+else

```

Comment:
wouldn't it be better to add `-lexecinfo` to `PcapPlusPlus.mk.freebsd`?

Reply:
Done. See my commit 3d8f2894b2.